cmd/mo-service/config.go

```diff
@@ -153,20 +156,61 @@ func (c *Config) validate() error {
 func (c *Config) createFileService(defaultName string) (*fileservice.FileServices, error) {
 	// create all services
 	services := make([]fileservice.FileService, 0, len(c.FileServices))
+
+	type perfCounterInfo struct {
+		fsName      string
+		perfCounter *perfcounter.Counter
```

Comment:
does this really need to be a pointer?   can it be a struct member?

Reply:
Done